.github/ISSUE_TEMPLATE.md

```diff
@@ -12,6 +12,11 @@
 
 **What is the [`ember-cli-rails-addon`][addon] version (from `package.json`)?**
 
+**What is your application server (e.g. `puma`, `unicorn`, `thin`, `webrick`,
+etc.)?**
+
+**Is your application server multi-threaded or multi-process?**
```

Comment:
Maybe.

To give you more background, `ember-cli-rails` will manage an `ember server` subcommand when run in development mode. 

Running on a multi-{process,thread} server, the gem would run an `ember server` subcommand for each worker {process,thread}, which would end up clobbering each other. This leads to a 1 in N (where N is the number of workers) chance that your request for EmberCLI-generated assets will be successful.

> (It could be that I still don't understand if this is the right question to ask.)

Did that explanation help? Does the question in the template feel like the right thing to ask?